android/settings.gradle

```diff
@@ -1,5 +1,6 @@
 include ':app-example'
-include ':templates'
+include ':templates:snapshot'
+include ':templates:release-0.3.1'
```

Comment:
Do we have to create a new module for each release?

Reply:
Yes, and it's intended. That's how we can maintain multiple versions that can also be compiled. Older versions would depend on binary sources of an older version of the library, while snapshot can rely on current project source.

Reply:
I think yes for now, for each release with changes in the templates